### ISSUE TITLE
Incorrect "Blank line between documentation and definition" when there is no documentation

### ISSUE THREAD

**Issue by mtancoigne** [mensfeld/yard-lint#300]
Hello!
The more I use yard-lint, the more my documentation improves, but I encounter a few issues:

Given this code:

```rb
    KEY_F11 = "\e[23~"
    # KEY_F12 = "\e[1~" # FIXME

    KEY_C_A = "\u0001"
    KEY_C_B = "\u0002"
    # ...
```

yard-lint raises this error:
```txt
[C] <the file path>:<line>
    : Blank line between documentation and definition for 'ATui::Input::KEY_C_A'
```

`Documentation/UndocumentedObjects` is disabled on the whole file because I won't document any of the constants here; blank lines are visual separators for large constants blocks.

While I understand the _why_ it's raised, it _feels_ incorrect; and there is no validator name provided by the error so I can't turn it off.

I see a few ways to improve the situation:
- have a way to disable the validator (fine by me) but it should have a name/have its name displayed
- have a way to allow comments separated by blank lines as they don't necessarily represent object documentation when documentation is
  disabled for some objects:
  ```rb
  # Does things
  def something; end
  
  # This comment is only here to clarify something in the class, addressed to the class devs and won't be part of the doc

  def something_else; end # documentation disabled here, so ignore comment position above?
  ```

**Comment by mensfeld** [comment]
fixed

**Comment by mtancoigne** [comment]
Thank you !
